datafusion/core/src/dataframe.rs

```diff
@@ -62,6 +62,7 @@ use crate::prelude::SessionContext;
 /// ```
 /// # use datafusion::prelude::*;
 /// # use datafusion::error::Result;
+/// # use datafusion::execution::context::Reader;
```

Comment:
I am somewhat worried about session context getting a new trait like this as now everyone who uses SessionContext must use now add this new `use` statement